feature-libs/order/components/order-details/order-details-extended/download-invoices/download-order-invoices-dialog.component.ts

```diff
@@ -0,0 +1,74 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import {
+  Component,
+  ChangeDetectionStrategy,
+  OnInit,
+  OnDestroy,
+  ViewChild,
+  ChangeDetectorRef,
+  AfterViewChecked,
+  inject,
+} from '@angular/core';
+import { Order, OrderOutlets } from '@spartacus/order/root';
+import { InvoicesListComponent } from '@spartacus/pdf-invoices/components';
+import { PDFInvoicesFacade } from '@spartacus/pdf-invoices/root';
+import {
+  ICON_TYPE,
+  FocusConfig,
+  LaunchDialogService,
+} from '@spartacus/storefront';
+import { Subscription } from 'rxjs';
+
+@Component({
+  selector: 'cx-download-order-invoices-dialog',
+  templateUrl: './download-order-invoices-dialog.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class DownloadOrderInvoicesDialogComponent
+  implements OnInit, OnDestroy, AfterViewChecked
+{
+  protected subscription = new Subscription();
+  @ViewChild(InvoicesListComponent, { static: false })
+  public invoiceComponent: InvoicesListComponent;
+  readonly OrderOutlets = OrderOutlets;
+  invoiceCount: number | undefined = undefined;
+  iconTypes = ICON_TYPE;
+  focusConfig: FocusConfig = {
+    trap: true,
+    block: true,
+    autofocus: true,
+    focusOnEscape: true,
+  };
+  order: Order;
```

Comment:
[MEDIUM]
I have problem to find usages of the `order` property in this component's TS and html. Is this property _really_ needed?

Reply:
not needed. have removed it.